src/TreeView/types.ts

```diff
@@ -17,8 +16,10 @@ export type INodeRefs = null | React.RefObject<{
 export type EventCallback = <T, E>(
   event: React.MouseEvent<T, E> | React.KeyboardEvent<T>
   ) => void;
-  
-  export interface INode<M extends IFlatMetadata = IFlatMetadata> {
+
+export type IMetadata = undefined;
+
+  export interface INode<M = IMetadata> {
```

Comment:
Typescript made a change to have any generic default to unknown, which I agree with quite a bit since it clarifies that you can’t know. So please make this INode<M = unknown>